packages/notification-services-controller/CHANGELOG.md

```diff
@@ -7,6 +7,24 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.0.0]
+
+### Added
+
+- added new public methods `enablePushNotifications` and `disablePushNotification` on `NotificationServicesController` ([#5120](https://github.com/MetaMask/core/pull/5120))
+- added `isPushEnabled` and `isUpdatingFCMToken` to `NotificationServicesPushController` state ([#5120](https://github.com/MetaMask/core/pull/5120))
+- added `/push-services/web` subpath export to make it easier to import web helpers ([#5120](https://github.com/MetaMask/core/pull/5120))
+
+### Changed
+
+- Optimized API calls for creating push notification links ([#5358](https://github.com/MetaMask/core/pull/5358))
+- Bump `@metamask/utils` from `^11.1.0` to `^11.2.0` ([#5301](https://github.com/MetaMask/core/pull/5301))
+- **BREAKING**: updated `NotificationServicesPushController` constructor config to provide an push interface ([#5120](https://github.com/MetaMask/core/pull/5120))
```

Comment:
```suggestion
- **BREAKING**: updated `NotificationServicesPushController` constructor config to require a push interface ([#5120](https://github.com/MetaMask/core/pull/5120))
- Optimized API calls for creating push notification links ([#5358](https://github.com/MetaMask/core/pull/5358))
- Bump `@metamask/utils` from `^11.1.0` to `^11.2.0` ([#5301](https://github.com/MetaMask/core/pull/5301))

Reply:
Fixed in https://github.com/MetaMask/core/pull/5378/commits/8a5b47caebcacfbca28cac56f682c198211d0083